src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -4742,13 +4742,30 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
                         ASRUtils::expr_value(enum_member_variable->m_symbolic_value));
             } else if (ASR::is_a<ASR::Module_t>(*t)) {
                 ASR::Module_t *m = ASR::down_cast<ASR::Module_t>(t);
-                std::string sym_name = value + "@" + x.m_attr;
+                std::string sym_name = value + "_" + x.m_attr + "_external__";
                 ASR::symbol_t *sym = current_scope->resolve_symbol(sym_name);
```

Comment:
Should we insert this in the global scope itself?

Reply:
Not needed IMO. Otherwise it would get declared scope in the C code as well and will mix things up with other scopes as well.